hack/deploy-test-appset.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
-set -ex
+set -e
 
 root=$(dirname "${BASH_SOURCE[0]}")
 
 prevcontext=$(kubectl config current-context)
 kubectl config use-context kind-argocd-control-plane
 
-# TODO: Make this generate argo apps in all created clusters
+kubectl get applications -n argocd --no-headers | cut -d' ' -f1 | xargs kubectl -n argocd delete application
```

Comment:
Have you tried `--all` in the `kubectl delete` ?

Reply:
Ah, good call, I kept it like this while testing the script as it allowed me to try and delete a single application at a time instead of nuking everything. I don't see why `kubectl delete --all` wouldn't work here. 👍